main.py

```diff
@@ -1,19 +1,30 @@
+# -*- encoding: utf-8 -*-
 import argparse
 from converter.converter import *
 
-
 CELSIUM_INPUT = ['celsius', 'cels', 'cel', '-c', 'c']
 FAH_INPUT = ['fahrenheit', 'fahren', 'far', '-f', 'f']
 QUIT_INPUT = ['quit', 'q', 'exit']
 
-
 parser = argparse.ArgumentParser()
 parser.add_argument("-v", "--verbose", help="It just return your argument")
 parser.add_argument("-f", "--fahrenheit", help="Take temperature in Fahrenheit and convert to Celsius", type=float)
 parser.add_argument("-c", "--celsius", help="Take temperature in Celsius and convert to Fahrenheit", type=float)
 
 args = parser.parse_args()
 
+
+def check_number(arg):
+    """
+    Check if the user's input convertible to float type
+    """
+    try:
+        float(arg)
+        return float(arg)
```

Comment:
@MariaMaximova, your method check if a string contains a float. Thus you have to return `True` here:
```suggestion
        return True
```

Reply:
Resolved in this [commit](https://github.com/MariaMaximova/convert_temp/pull/9/commits/ea07c0df2d274ab51fb7e4346d05465b52c56248)